<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostHomeWidget</class>
 <widget class="QWidget" name="HostHomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>266</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/image/HostHomeUI/bg_home.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="hostNameLb">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>61</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="popNumLb">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>84</y>
      <width>63</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fbd52d;
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="popTextLb">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>83</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>本周热度</string>
    </property>
   </widget>
   <widget class="QLabel" name="incomeTextLb">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>105</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>本月收益</string>
    </property>
   </widget>
   <widget class="QLabel" name="incomeNumLb">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>103</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#incomeNumLb{color:#fbd52d; font: 75 12px &quot;宋体&quot;;}
QToolTip{border-width: 1px;border-image: url(://image/ToastWidget/toast.png) 2 2 2 2 stretch stretch;color:white;}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0 &lt;span style=&quot; color:#ffffff;&quot;&gt;元&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>179</y>
      <width>139</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{border-image:url(:/image/HostHomeUI/bt_start_hover.png);}
QPushButton{border-image: url(:/image/HostHomeUI/bt_start.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adviceBtn">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>11</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/HostHomeUI/bt_useradvice_hover.png);}
QPushButton{border-image: url(:/image/HostHomeUI/bt_useradvice.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover{border-image:url(:/image/HostInviteUI/bt_help_hover.png);}
QPushButton{border-image: url(:/image/HostInviteUI/bt_help.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/HostHomeUI/bt_close_hover.png);}
QPushButton{border-image: url(:/image/HostHomeUI/bt_close.png);}



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="viewBtn">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>51</y>
      <width>324</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/HostHomeUI/img_banner.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgTextLb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>212</y>
      <width>285</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 75 12px &quot;宋体&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:12px; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;主播 &lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; color:#ffd32b;&quot;&gt;兔子一小团&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt; 昨日&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; color:#ffd32b;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;次被邪恶的玩家变成了汤姆&lt;br /&gt;猫情场十分得意！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/HostHomeUI/bt_min_hover.png);}
QPushButton{border-image: url(:/image/HostHomeUI/bt_min.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="loadingWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>266</height>
    </rect>
   </property>
   <widget class="QLabel" name="loadingLb">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>118</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
